### DISCUSSION TITLE
is_conversational_answer is removed from PandasAI class.

### DISCUSSION THREAD

**Discussion by asharali95** [sinaptik-ai/pandas-ai#518] [Q&A]
In latest version of pandas AI, I am unable to use is_conversational_answer param from run() in PandasAI class. Now, the response I am getting does not feel natural. What should I do now?

**Comment by mspronesti** [comment]
I believe that feature is no longer supported but a new replacement is about to be implemented, as explained [here](https://github.com/gventuri/pandas-ai/discussions/494#discussioncomment-6857477).
Also, please notice that `PandasAI` (class) is deprecated and will be removed soon. From pandasai v1.0 onwards, you should use `pandasai.SmartDataframe` instead.
